src/ansys/geometry/core/designer/body.py

```diff
@@ -818,6 +858,25 @@ def rotate(
             )
         )
 
+    @protect_grpc
+    @check_input_types
+    @reset_tessellation_cache
+    @min_backend_version(24, 1, 0)
```

Comment:
Oh! It was already available for all backends?

Reply:
@RobPasMue No, but i was able to retroactively add it, so it is technically available with 24.1 but they would need an updated service :)

Reply:
In that case... let's avoid "retroactive" behavior. I talked about this with @b-matteo. Your service is the one that defines the backend version number so you need at least a 24R2 service to do this. Meaning that we need to add the decorators as 24, 2, 0

Reply:
@RobPasMue makes sense, on it